index.html

```diff
@@ -0,0 +1,240 @@
+          <div class="news__heading">
+            <h5>What's Happening</h5>
+          </div>
+          <div class="news-item news-item-1__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-2__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-3__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-4__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="image-container__news-1">
+            <img
+              class="boolean-logo"
+              src="../boolean-uk-html-boolean-image-grid/assets/images/boolean.jpeg"
+              alt="boolean-logo"
+              height="50px"
+              width="50px"
+            />
+          </div>
+          <div class="image-container__news-2">
+            <img
+              class="boolean-logo"
+              src="../boolean-uk-html-boolean-image-grid/assets/images/boolean.jpeg"
+              alt="boolean-logo"
+              height="50px"
+              width="50px"
+            />
+          </div>
+          <div class="image-container__news-3">
+            <img
+              class="boolean-logo"
+              src="../boolean-uk-html-boolean-image-grid/assets/images/boolean.jpeg"
+              alt="boolean-logo"
+              height="50px"
+              width="50px"
+            />
+          </div>
+          <div class="image-container__news-4">
+            <img
+              class="boolean-logo"
+              src="../boolean-uk-html-boolean-image-grid/assets/images/boolean.jpeg"
+              alt="boolean-logo"
+              height="50px"
+              width="50px"
+            />
+          </div>
+        </div>
+      </div>
+      <!-- main section -->
+      <div class="main">
+        <div class="main__top">
+          <div class="main__top-a">
+            <div class="box-container box-container--left">
+              <h4>Home</h4>
+            </div>
+            <div class="box-container box-container--right">
+              <i class="far fa-star"></i>
+            </div>
+          </div>
+          <div class="main__top-b">
+            <div class="main__top-b-pic">
+              <img
+                class="avatar"
+                src="../boolean-uk-html-twitter/images/avatar-one.jpg"
+                alt=""
```

Comment:
It's important to have an `alt` field for every image we use, that way people using devices such as screen-readers can understand what our images are. I believe this is for a profile, so in this case it would be acceptable to have the user's twitter handle/full name if available.